partiql-tests-data/eval/query/pivot.ion

```diff
@@ -45,12 +45,17 @@ pivot::[
   {
     name:"pivotBadFieldType",
     statement:"PIVOT a.name AT i FROM animals AS a AT i",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:{
+    assert: [
+      {
+        evalMode: EvalModeCoerce,
+        result: EvaluationSuccess,
+        output:{}
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail
```

Comment:
- [X] Fails because it attempts to create tuples like `{0: 'Kumo'}` and `0` is an invalid type for the key of a struct.